app/actions/posts.js

```diff
@@ -84,3 +83,84 @@ export const createPost = (title, description, imageData) => {
     })
   };
 };
+
+
+export const likePost = (postId) => {
+  return (dispatch, getState) => {
+    const currentUser = getState().currentUser
+
+    dispatch({
+      type: 'LOAD_SPINNER'
+    });
+
+    return fetch(URL + 'posts/' + postId + '/like', {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'X-User-Email': currentUser.email,
+        'X-User-Token': currentUser.authentication_token,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        like: {
+          post: postId,
+          user: currentUser.id
```

Comment:
The server should determine this. Because if I make this request manually I could make *any* user like *any* post.